corehq/apps/export/models/new.py

```diff
@@ -853,7 +853,8 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
             app_id,
             current_schema.last_app_versions,
         )
-        app_build_ids.append(app_id)
+        if app_id:
```

Comment:
@NoahCarnahan now if there is no app_id included in the case export for something like `commcare-user`, then weird things won't happen. also removed the `app_id` parameter below for that reason and that it would not properly parse case_types with multiple apps